config/autoload/local.php.dist

```diff
@@ -1,3 +1,9 @@
 <?php
+
+use Zend\ConfigAggregator\ConfigAggregator;
+
 return [
+    // Cache the configuration: Recommended for production
+    // Closures used somewhere in the configuration make caching impossible
+    ConfigAggregator::ENABLE_CACHE => false,
```

Comment:
Why is this in local.php.dist? I would think development.config.php.dist is the right place for this?